src/dmd/cparse.d

```diff
@@ -4557,6 +4571,35 @@ final class CParser(AST) : Parser!AST
         return s;
     }
 
+    /********************************
+     * Determines if type t is a function type.
+     * Make this work without needing semantic analysis.
+     * Params:
+     *  t = type to test
+     * Returns:
+     *  true if it represents a function
+     */
+    bool isfunction(AST.Type t)
```

Comment:
Would prefer camel-casing the function name, but not a big deal.  Also, `private`?

Reply:
done